<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchDB</class>
 <widget class="QDialog" name="searchDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Искать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Печать</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Montserrat Light</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Поиск:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="font">
       <font>
        <family>Montserrat Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>NameAction</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>221</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_4">
         <property name="text">
          <string>id_book</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBox_5">
         <property name="text">
          <string>count</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBox_8">
         <property name="text">
          <string>address</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_7">
         <property name="text">
          <string>fio</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_6">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="7" column="0">
        <widget class="QLineEdit" name="lineEdit_12"/>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_9">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_10">
         <property name="text">
          <string>name</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLineEdit" name="lineEdit_13">
         <property name="text">
          <string>0</string>
         </property>
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBox_11">
         <property name="text">
          <string>id_author</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="checkBox_12">
         <property name="text">
          <string>id_typeStyle</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="checkBox_13">
         <property name="text">
          <string>year</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="checkBox_14">
         <property name="text">
          <string>countList</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLineEdit" name="lineEdit_14">
         <property name="text">
          <string>0</string>
         </property>
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="lineEdit_11"/>
       </item>
       <item row="12" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_5">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBox_17">
         <property name="text">
          <string>date</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_15">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_16"/>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="lineEdit_18"/>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_15"/>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_16">
         <property name="text">
          <string>nameAction</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="checkBox_18">
         <property name="text">
          <string>nameTable</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="lineEdit_17"/>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_6">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="9" column="0">
        <widget class="QLineEdit" name="lineEdit_23"/>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="checkBox_23">
         <property name="text">
          <string>id_book</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_20"/>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBox_21">
         <property name="text">
          <string>id_action</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLineEdit" name="lineEdit_25">
         <property name="text">
          <string>0</string>
         </property>
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_19"/>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_19">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="lineEdit_21"/>
       </item>
       <item row="12" column="0">
        <widget class="QCheckBox" name="checkBox_25">
         <property name="text">
          <string>price</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="checkBox_22">
         <property name="text">
          <string>id_staffMember</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLineEdit" name="lineEdit_24">
         <property name="text">
          <string>0</string>
         </property>
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="lineEdit_22"/>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="checkBox_24">
         <property name="text">
          <string>count</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_20">
         <property name="text">
          <string>date</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_7">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="7" column="0">
        <widget class="QLineEdit" name="lineEdit_29">
         <property name="maxLength">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBox_28">
         <property name="text">
          <string>numberPhone</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="lineEdit_28">
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_26">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_27">
         <property name="text">
          <string>fio</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="checkBox_29">
         <property name="text">
          <string>inn</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_27"/>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_26"/>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_8">
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_30">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_31">
         <property name="text">
          <string>nameStyle</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_30"/>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_31"/>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
